<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="func1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Функия 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="func2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Функция 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="x0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начальная точка</string>
   </property>
  </widget>
  <widget class="QLabel" name="nu1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметр 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="a0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Коэффициент alp0</string>
   </property>
  </widget>
  <widget class="QLabel" name="b0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Коэффициент bet0</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>20</width>
     <height>911</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>821</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>771</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="set_values">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Вычисление</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>590</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="a1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Коэффициент alp1</string>
   </property>
  </widget>
  <widget class="QLabel" name="b1_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Коэффициент bet1</string>
   </property>
  </widget>
  <widget class="QLabel" name="req">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>761</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nu2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметр 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="B0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение B</string>
   </property>
  </widget>
  <widget class="QLabel" name="A0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение А</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alp0_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bet0_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bet1_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="x0_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="B_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f2_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alp1_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu2_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f1_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="A_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu1_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="a_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_gr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Левый конец интервала</string>
   </property>
  </widget>
  <widget class="QLabel" name="r_gr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Правый конец интервала</string>
   </property>
  </widget>
  <widget class="QLabel" name="accuar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Точность</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accur_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
